tests/PHPStan/Analyser/data/bug-8467b.php

```diff
@@ -0,0 +1,56 @@
+<?php declare(strict_types = 1);
+
+namespace Bug8467b;
+
+use function PHPStan\Testing\assertType;
+
+class Test {
+	public function foo (?string $cwd, bool $initialClone = false): void {
+		if ($initialClone) {
+			$origCwd = $cwd;
+			$cwd = null;
+		}
+
+		if ($initialClone && isset($origCwd)) {
+			assertType('string', $origCwd);
+			assertType('null', $cwd);
```

Comment:
I believe this one is not covered in `createConditionalExpressions` yet and not working in 1.9.2 too.
I think it might be better to solve in a different PR